continuous_delivery/deploy_and_basics.rst

```diff
@@ -80,26 +77,30 @@ Deploy a Specific Git Tag
 
 **Build branch** allows you to specify to deploy an arbitrary Git branch or tag.
 
-.. important::
-   #. This should only be used with test systems [#f1]_.
-   #. The value of **Build branch** is ignored if `Docker Image ID <#run-menu-advanced>`__ is set.
+.. note:: There might be a situation where you want to deploy a tag directly on production. 
+          In that case remove the CD parameter "DOCKER_PULL_URL". `Deploy s Specific Docker Image <#deploy-a-specific-docker-image>`_.
 
 
-Deploy a Specific Docker Image Tag
-----------------------------------
+Deploy a Specific Docker Image
+------------------------------
 
-This is only used for production systems, it allows you to deploy a docker image that was once installed on the test
-system. By default the ``test`` tag is deployed which installs the very same image the test system is using. You can
-however also install an older image formerly installed on the test system.
+This is used for production systems, it allows you to deploy a docker image that was once installed on the test
+system. The image is determined by the DOCKER_PULL_URL parameter. It allows you to pick a image from a project an deploy it on your target project.
+For this you can fill the parameter with the following: 
 
-#. Find the tag name for the version you want to have installed. It looks like this ``ecap-2017-05-09T12_54_27``.
-   **ecap** is the customer name.
-#. Enter the tag in the **Docker Image ID** field in the `run menu <#run-menu-advanced>`__.
+**registry.appuio.ch/toco-nice-%env.INSTALLATION%test/%env.DOCKER_IMAGE%.**
 
-.. todo:: describe where the tag names can be found
+After the evaluation of the CD parameters it will appear as an URL like this: 
+         
+``registry.appuio.ch/tocco-nice-tocco/nice:latest`` [#f1]_
```

Comment:
Hard to understand, I'd add an example. E.g. this deploys the image from toccotest if you're deploying tocco. Shouldn't it be tocco**test** in the URL?